src/components/add-tag-dialog/add-tag-dialog.js

```diff
@@ -9,7 +9,8 @@ export class AddTagDialogComponent {
     this.props = props;
   }
 
-  showDialog() {
+  showDialog(tagsValue) {
+    this.tagsValue = tagsValue;
```

Comment:
`tagsValues` is an array -- so this name should be in plural -- `tagsNames` -- denoting that this is array. Also you can make it more clear to others by naming it `existingTagNames`